### DISCUSSION TITLE
Math, implementation and accurate interpretation of `monai.transforms.Rand3DElastic`

### DISCUSSION THREAD

**Discussion by YerePhy** [Project-MONAI/tutorials#763] [Q&A] [answered]
Hi, 

I've search about elastic deformations, in the computer science context not in physics context, but I haven't find a detailed description of how this tranformations are done and the intiuitive idea behind. If someone can explain at least the intuition and point out some references and explain each parameter of `monai.transforms.Rand3DElastic` (the non-trivial ones) it would be very helpful, or at least a good guess for the parameters.

Thanks.

**Comment by wyli** [comment, marked as answer]
sure, this paper describes Gaussian filtering over the random displacement fields, it visually resembles elastic deformation https://www.microsoft.com/en-us/research/wp-content/uploads/2003/08/icdar03.pdf

**Reply by YerePhy** [reply]
Thanks @wyli!
